src/ZeroconfClient.ts

```diff
@@ -13,6 +13,6 @@ import { Bonjour, Browser } from 'bonjour'
 export function supplyClient(bonjour: Bonjour, client: Client): Browser {
 	const browser = bonjour.find({ type: SERVICE_TYPE })
 	browser.on('up', (service) => client.introduce(service.host, service.port))
-	client.once('done', browser.stop)
+	client.once('done', () => browser.stop())
```

Comment:
Why create a new function? Why not just use `browser.stop` or `browser.stop.bind(browser)`?

Reply:
They're banned by the style guide 😆. I don't mind either way, good to have it consistent though.